### PR TITLE
[CENT-392] write README for MINT p0

### PR DESCRIPTION
**Pull request by mirathewhite**
The file doc/masterminter.md is a README file explaining the usage and functionality of the MasterMinter contract.

### CODE REVIEW

File: doc/masterminter.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# MasterMinter contract
+The MasterMinter is a governance contract.  It delegates the functionality
+of the `masterMinter` role in the Centre USDC contract to multiple addresses.
```
Comment (o-a-hudson) [inline doc/masterminter.md]: I think we always use the upper case of CENTRE.

File: doc/masterminter.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  perform any actions on the MasterMinter contract.  It interacts only with
+  the FiatToken contract.
+
+# Interaction with FiatToken contract
+The `owner` of the FiatToken contract can set the `masterMinter` role to point
+to the address of the `MasterMinter` contract.  This enables the `MasterMinter`
+contract to call minter management functions on the FiatToken contract:
+
+- `configureMinter(minter, allowance)` - Enables the `minter` and sets its minting allowance.
+- `removeMinter(minter)` - Disables the `minter` and sets its mintingallowance to 0.
```
Comment (o-a-hudson) [inline doc/masterminter.md]: "mintingallowance" looks like a typo.

File: doc/masterminter.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- <b>AllowanceController.</b> Use this `controller` to enable the `minter` with a single
+ `configureMinter` transaction, and then use it exclusively to sign `incrementMinterAllowance` transactions.
+ There may be multiple `AllowanceControllers` that sign different size allowance increment transactions.
+- <b>SecurityController.</b> Use this `controller` to sign a single `removeMinter` transaction and store
+  it for emergencies.
+
+The private keys to the `AllowanceController` and `SecurityController` should stay in cold storage.
+This configuration lets the Controller keep multiple warm `incrementMinterAllowance` transactions on hand,
+as well as the `removeMinter` transaction in case of a problem.  Broadcasting the `removeMinter` transaction
+will cause all future `incrementMinterAllowance` transactions to `throw`.  Since there two types
```
Comment (o-a-hudson) [inline doc/masterminter.md]: Looks like there should be "the"

File: doc/masterminter.md (hunk 1 above)
Comment (mirathewhite) [reply doc/masterminter.md]: fixed

File: doc/masterminter.md (hunk 2 above)
Comment (mirathewhite) [reply doc/masterminter.md]: fixed

File: doc/masterminter.md (hunk 3 above)
Comment (mirathewhite) [reply doc/masterminter.md]: fixed

**Comment by mirathewhite** [comment]
Fixed all typos detected by @o-a-hudson

**Pull request merged**